pages/restaurant/[id].js

```diff
@@ -0,0 +1,94 @@
+import { connectToDatabase } from "../../util/mongodb";
+import { ObjectId } from "mongodb";
+import { Typography, Rate, Divider } from "antd";
+import Head from "next/head";
+
+import BasicInfo from "../../components/restoprofile/BasicInfo";
+import ImageHeader from "../../components/restoprofile/ImageHeader";
+import Reviews from "../../components/restoprofile/Reviews";
+import Gallery from "../../components/restoprofile/Gallery";
+
+import styles from "../../styles/restoprofile/restaurantprofile.module.css";
+
+const { Title } = Typography;
+
+export default function RestaurantProfile({ resto }) {
+  return (
+    <div className={styles.wrapper}>
+      <Head>
+        <title>{resto.name}</title>
+      </Head>
+      <div className={styles.contentContainer}>
+        <ImageHeader />
+
+        <div className={styles.contentBody}>
+          <div>
+            <Title>{resto.name}</Title>
+            <Rate allowHalf value={resto.averageRating} disabled />
+            &nbsp;&nbsp;&nbsp;
+            {resto.averageRating}
+          </div>
```

Comment:
Let's comment this out first since we don't have reviews yet

Reply:
We do have ratings